packages/analytics/src/helpers.ts

```diff
@@ -304,7 +304,8 @@ function wrapGtag(
           gtagParams as GtagConfigOrEventParams
         );
       } else if (command === GtagCommand.CONSENT) {
-        const [gtagParams] = args;
+        // If CONSENT, second arg must be 'default' or 'update'
```

Comment:
I don't think we need this comment, it looks like the issue is that it was passing the method instead of the params as the third argument, not that "update" was missing.